Sources/ArcGISToolkit/Components/Compass/Compass.swift

```diff
@@ -35,7 +35,11 @@ public struct Compass: View {
     /// The opacity of the compass.
     @State private var opacity: Double = .zero
     
-    /// A Boolean value indicating whether  the compass should automatically
+    /// A Boolean value indicating whether sensory feedback is enabled
+    /// when the heading snaps to zero.
+    private var snapToZeroSensoryFeedbackEnabled: Bool = false
```

Comment:
I'm not sure why the `action` property is out of order, but my thinking is that it is best we keep these properties alphabetized. Should we move this down below `size` and move `action` up above `autoHide`?